gaelib/view/base_view.py

```diff
@@ -84,12 +91,71 @@ def validate_request_args(self):
 
   def validation_error(self):
     if self.missing_args:
-      return self.json_error("Missing Arguements: " + str(self.missing_args), 200)
+      return self.json_error("Missing Arguments: " + str(self.missing_args), 200)
 
 
-class BaseApiHandler(BaseHttpHandler):
+class BaseAPIHandler(BaseHttpHandler):
+  controller = None
   decorators = [auth_required]
 
+  def validate(self, entity_id):
+    if not self.controller:
+      error = "No controller defined"
+      response, _, _ = self.json_error(error, 500)
+      raise HTTPException(error, response)
```

Comment:
HTTPException expects the return to be of type `Response | None`. Here we are returning a response dict. This did raise an error during testing with the package in cm-facade-jira

Reply:
I've decided to take up all lint related fixes in another pull request. So punting this to then.